VTUL/dlp-access
# Add 3D/2D view

**Adds view to render x3dom model and mirador viewer in same display and allow toggling between them.**
* * *

**JIRA Ticket**: (link) (:star:)

* Other Relevant Links (Meeting note, project page, related pull requests, etc.)

# What does this Pull Request do? (:star:)
Adds view to render x3dom model and mirador viewer in same display and allow toggling between them.

# What's the changes? (:star:)
* Adds new view option that contains:
    * x3dom
    * mirador
    * leaflet thumbnail w/ location displayed

# How should this be tested?
* View 3d/2d record here: https://3d-view-refactor.d2ysrrdhih4bgc.amplifyapp.com/archive/j636089g
* Check that 3d model loads
* Check that leaflet thumbnail loads with correct content.
* Check that you can click "View 2D Full Size" link and it will toggle to Mirador view
* Check that you can toggle back and forth between views
* Check other records to make sure that iiif only records (no 3d) are still functional

**Already imported 3D/2D records include:**
VTEC000004835
VTEC000004836
VTEC000004837
VTEC000004838
VTEC000004841
VTEC000004842
VTEC000004843
VTEC000004844
VTEC000004845
VTEC000004847
VTEC000004848
VTEC000004849
VTEC000004850
VTEC000004851
VTEC000004853
VTEC000004854



# Additional Notes:
* branch: `3d_view_refactor`

# Interested parties
@goynejennifer @otokama 

(:star:) Required fields
